packages/manager/apps/telecom/package.json

```diff
@@ -34,8 +34,8 @@
     "@ovh-ux/manager-telecom-dashboard": "^5.0.0",
     "@ovh-ux/manager-telecom-styles": "^3.1.0",
     "@ovh-ux/manager-telecom-task": "^5.0.0",
-    "@ovh-ux/ng-at-internet": "^5.0.0",
-    "@ovh-ux/ng-at-internet-ui-router-plugin": "^3.0.0",
+    "@ovh-ux/ng-at-internet": "^4.0.4",
+    "@ovh-ux/ng-at-internet-ui-router-plugin": "^2.0.4",
```

Comment:
Hmm you need to downgrade those packages?

Reply:
Oups, I update this now.